sql/core/pom.xml

```diff
@@ -86,15 +86,17 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-storage-api</artifactId>
+    </dependency>
```

Comment:
BTW, you should not introduce `hive` into `sql/core`.
Apache ORC should be used as Apache ORC.